client/src/components/timeline/revision.js

```diff
@@ -7,6 +7,32 @@ import Wiki from 'app/entities/wiki';
 import Date from './date';
 import Timelapse from './timelapse';
 
+// eslint-disable-next-line no-useless-escape
+const REGEX_EDIT_SUMMARY_PARTS = /(?:\/\*([^\*]+)\*\/)?(.+)?/;
```

Comment:
what part of that is violating [no-useless-escape](https://eslint.org/docs/rules/no-useless-escape)?

Reply:
If I had to guess it gets confused on ...[^\*]+...

Reply:
I think I don't need to scape `*` there so I'll fix it